playground/src/Playground.tsx

```diff
@@ -40,7 +40,7 @@ export const Playground = ({ code, scope }: PlaygroundProps) => (
     code={code.trim()}
     scope={scope}
     theme={oceanicNext}
-    transformCode={(code) => `<>${code}</>`}
+    transformCode={(code) => (code.startsWith('<') ? `<>${code}</>` : code)}
```

Comment:
add a trim to code in case of

Reply:
fixed https://github.com/divriots/dockit-react/pull/54